.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: ci
+
+on:
+  push:
+  pull_request:
+jobs:
+  run-tests:
+    name: Run Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version:
+          - 1.18
+          - 1.17
+          - 1.16
+          - 1.15
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: Install deps 
+        run: go get -d ./...
```

Comment:
I thought I commented on this in my previous review, but I guess not. Suggestion would be to make this `go mod download`